prebuilt-tasks/pom.xml

```diff
@@ -77,5 +77,17 @@
             <groupId>org.assertj</groupId>
             <artifactId>assertj-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>dev.parodos</groupId>
+            <artifactId>notification-service-sdk</artifactId>
+            <version>${revision}</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-web</artifactId>
+            <version>5.3.15</version>
```

Comment:
It would be good to use the latest [version](https://mvnrepository.com/artifact/org.springframework/spring-web/5.3.26) from this stable channel

Reply:
Done